### PR TITLE
Support listing Magnum bays

### PR DESCRIPTION
**Pull request by carolynvs**
For #10.

- [x] GET /bays
- [x] Unit tests
- [x] Acceptance tests

### CODE REVIEW

**Comment by jrperritt** [comment]
Ready for review? Feel free to ~~slap~~ ping me if I'm slacking on reviews.

**Comment by carolynvs** [comment]
Doh! Sorry yes this is ready for review. 😄

File: openstack/containerorchestration/v1/bays/requests.go [hunk 1]
```diff
@@ -2,8 +2,49 @@ package bays
...
+}
+
+// ToBayListQuery formats a ListOpts into a query string.
+func (opts ListOpts) ToBayListQuery() (string, error) {
+	q, err := gophercloud.BuildQueryString(opts)
+	return q.String(), err
+}
+
+// List returns a Pager which allows you to iterate over a collection of
+// bays. It accepts a ListOpts struct, which allows you to sort
```
Comment (jrperritt) [inline openstack/containerorchestration/v1/bays/requests.go]: s/ListOpts struct/`ListOptsBuilder`

**Comment by jrperritt** [comment]
Just that ^

**Comment by carolynvs** [comment]
@jrperritt I've fixed the docstring.

After you merge this, would you please update the magnum branch with the latest from master? It has that v3 auth scope fix. 😄

**Comment by jrperritt** [comment]
+2

**Pull request merged**